src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final float MULTIPLIER = 1.5f;
+    private int maxElements = 10;
+    private int size = 0;
```

Comment:
`int type = 0` by default
```suggestion
    private int size;
```